inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -125,7 +125,7 @@
             # If an exception occurs during string conversion, raise that exception
             raise SotaError('package_list is not a string in manifest') from e
 
-        self._device_reboot = parsed_manifest['deviceReboot']
+        self._reboot_device = parsed_manifest['deviceReboot']
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** The variable name _device_reboot has been changed to _reboot_device. Ensure that all references to this variable are updated accordingly throughout the codebase to maintain consistency.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self._reboot_device = parsed_manifest['deviceReboot']
```